tests/output_filters/virustotal/lookup_domains_test.py

```diff
@@ -13,3 +13,57 @@ def test_no_domains(self):
         ]
 
         self.run_test(LookupDomainsFilter, input_blobs=input_blobs, expected_output_blobs=input_blobs)
+
+    def test_benign_domains(self):
+        input_blobs = [
+            {'osxcollector_domains': ['good.example.com'], 'dingo': 'bingo', 'apple': [3, 14]},
+            {'osxcollector_domains': ['good.example.co.uk'], 'bingo': 'bongo', 'orange': 'banana'}
+        ]
+
+        self.run_test(LookupDomainsFilter, input_blobs=input_blobs, expected_output_blobs=input_blobs)
+
+    def test_suspicious_domains(self):
+        input_blobs = [
+            {'osxcollector_domains': ['evil.example.com'], 'dingo': 'bingo', 'apple': [3, 14]},
+            {'osxcollector_domains': ['evil.example.co.uk'], 'bingo': 'bongo', 'orange': 'banana'}
+        ]
+        output_blobs = [
```

Comment:
Using self. assert_key_added_to_blob would allow you make this dict a bit shorter like in 
https://github.com/Yelp/osxcollector/blob/master/tests/output_filters/find_blacklisted_test.py#L52